### PR TITLE
Phase II.F: E2E tests for currency info via ERC20 Minter

### PR DESCRIPTION
**Pull request by ryley-o**
## Description of the change

Add e2e tests for updating of currency info on a minter via the new ERC20V5 shared minter.

Note: Since the ERC20 minters themselves (Set Price, Merkle, and Holder variants) conform to the existing interfaces, no new handler code is required in this PR. The only thing updated here is to test out the functionality unlocked in the ERC20V5 Set Price minter. The assumption is that the contracts repo is responsible for testing that contracts emit appropriate events, and subgraph is simply responsible for testing that emitted events are interpreted correctly.

Note: try/catch blocks are cleaned up a bit in this PR across all tests, based off of some feedback in the PR 🙏

### CODE REVIEW

File: tests/e2e/runner/__tests__/minter-suite-v2/i-shared-minter-v0.test.ts [hunk 1]
```diff
@@ -119,17 +129,59 @@ describe("iSharedMinterV0 event handling", () => {
...
-  // });
+  describe("ProjectCurrencyInfoUpdated", () => {
+    afterEach(async () => {
+      // clear the minter for project zero
+      try {
+        await sharedMinterFilterContract
+          .connect(artist)
+          .removeMinterForProject(0, genArt721CoreAddress);
+      } catch (error) {
+        // swallow error in case of test failure
```
Comment (jakerockland) [inline tests/e2e/runner/__tests__/minter-suite-v2/i-shared-minter-v0.test.ts:140]: swallow error in case of test failure

Comment (jakerockland) [inline tests/e2e/runner/__tests__/minter-suite-v2/i-shared-minter-v0.test.ts:140]: @ryley-o why do we just swallow this error?

Comment (ryley-o) [reply tests/e2e/runner/__tests__/minter-suite-v2/i-shared-minter-v0.test.ts:140]: If the test failed for some reason, project 0 might not have a minter assigned, and the try block here might fail. Ideally, we will only see the failed test, and won't also see more information about a failed call in the afterEach.

**Comment by ryley-o** [comment]
gentle reminder here @jakerockland @yoshiwarab

**Comment by jakerockland** [comment]
> gentle reminder here @jakerockland @yoshiwarab

Thank you for the ping ser – TAL today!

**Review by jakerockland — APPROVED** [review]

File: tests/e2e/runner/__tests__/minter-suite-v2/i-shared-minter-v0.test.ts (hunk 1 above)
Comment (ryley-o) [reply tests/e2e/runner/__tests__/minter-suite-v2/i-shared-minter-v0.test.ts]: yes - updated with more comments and cleaned up some of the try/catch blocks across all of our e2e tests in [2d0cd26](https://github.com/ArtBlocks/artblocks-subgraph/pull/235/commits/2d0cd260b529c307c6e10beb4abdb882d1fa04f1)

**Review by yoshiwarab — APPROVED** [review]

**Pull request merged**